UPGRADE-GUIDE.md

```diff
@@ -0,0 +1,1298 @@
+---
+title: Upgrade guide
+section: overview
+---
+
+Hey Fliers, we've been busy for the past 12 weeks working on significant changes to PatternFly's HTML and CSS. We've changed colors, the default font, and made our components mobile-first. This upgrade guide details **what** was broken and **how** to fix it. We've included links to the pull requests in case you want to follow a trail of **why** a change was made.
+
+## Global
+### Colors
+Various colors have been updated to increase contrast in more usage scenarios to meet accessibility requirements. Some color variables have been remapped to better suit new component designs such as alerts, and we’ve also added new `-50` color tints to serve as colored backgrounds where appropriate. As a followup to last year's color changes, we've updated Blue and Cyan palettes' `-600` and `-700` values as well.
+
+### Mobile first CSS
+We've updated some components' CSS to be mobile first by using `@min-width` media queries. The components that are now mobile-first are: background image, data list, toolbar, form, login, page, toolbar, and wizard [(#2816)](https://github.com/patternfly/patternfly/pull/2816).
+  * All components are now mobile-first, but one component that cannot satisfy this is the table, specifically with use of the `.pf-m-grid[-on-{breakpoint}]` modifers. At the specified grid breakpoint, native/default properties are modified to achieve a grid based layout. To overwrite these changes would require adding back native properties to the table. For that reason, the table will remain the exception to a “mobile-first” approach.
+
+### Vertical navigation hidden breakpoint
+We've changed the breakpoint at which we hide the vertical nav to be `$pf-global--breakpoint--xl` (1200px) rather than `$pf-global--breakpoint--md` (768px) [(#2962)](https://github.com/patternfly/patternfly/pull/2962). The overall page chrome and some individual components' horizontal spacing becomes more compact at this breakpoint (true of the old breakpoint and the new), so applications may find that custom elements that aligned with the old breakpoint may not align now since the chrome spacing changes at 1200px instead of 768px. An application may need to make updates to match the chrome's spacing at the new breakpoint.
```

Comment:
```suggestion
We've changed the hidden breakpoint for the vertical nav to be `$pf-global--breakpoint--xl` (1200px) rather than `$pf-global--breakpoint--md` (768px) [(#2962)](https://github.com/patternfly/patternfly/pull/2962). The overall page chrome and some individual components' horizontal spacing becomes more compact at this breakpoint (true of the old breakpoint and the new). Applications' custom elements that aligned with the old breakpoint may not align now since the chrome spacing changes at 1200px instead of 768px. You may need to make updates to match the chrome's spacing at the new breakpoint.
```